src/schema/v2/artwork/artworkContextGrids/__tests__/auctionContext.test.ts

```diff
@@ -148,4 +148,37 @@ describe("Default Context", () => {
     // Related artworks grid should have no artworks
     expect(data.artwork.contextGrids[1].artworksConnection).toEqual(null)
   })
+
+  it("allows related artworks to be excluded", async () => {
+    context.relatedLayersLoader = () => Promise.resolve([{ id: "main" }])
+    context.relatedLayerArtworksLoader = jest.fn()
+
+    const response = await runAuthenticatedQuery(
+      gql`
+        {
+          artwork(id: "donn-delson-space-invader") {
+            contextGrids(includeRelatedArtworks: false) {
+              title
+              artworksConnection(first: 2) {
+                edges {
+                  node {
+                    slug
+                    title
+                  }
+                }
+              }
+            }
+          }
+        }
+      `,
+      context
+    )
+
+    const grids = response.artwork.contextGrids
+    const gridTitles = grids.map((grid) => grid.title)
+
+    expect(grids.length).toEqual(1)
+    expect(gridTitles).not.toInclude("Related works")
+    expect(context.relatedLayerArtworksLoader).not.toHaveBeenCalled()
+  })
```

Comment:
Nice! I like how this spec makes it very explicit and clear how we can exclude the relatedArtworks 👏